CMakeLists.txt

```diff
@@ -8,6 +8,8 @@
 # Check IF cmake has the required version
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11 FATAL_ERROR)
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
```

Comment:
I would actually drop this and leave it to the user. Notice that if you use alibuild this will nevertheless set to TRUE.